packages/components/src/components/alert/alert.tsx

```diff
@@ -14,6 +15,13 @@ export class alert {
   @Prop() size?: string = '';
   @Prop() theme?: string = '';
   @Prop() variant?: string = '';
+  @Prop({reflectToAttr: true}) title: string;
+  @Prop({reflectToAttr: true}) opened: boolean;
+  @Prop() timeout?: number;
```

Comment:
Please consider adding the defaultTimeout to the prop itself: `@Prop() timeout?: number = 500`;